flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphRestartTest.java

```diff
@@ -608,12 +569,11 @@ public void run() {
 
```

Comment:
Do we still need these wait calls?

Reply:
I think all wait related things can go away.